contracts/src/v0.8/automation/dev/test/WETH9.sol

```diff
@@ -0,0 +1,93 @@
+// Submitted for verification at Etherscan.io on 2017-12-12
+
+// Copyright (C) 2015, 2016, 2017 Dapphub
+
```

Comment:
this would probably be better in `vendor/` and can we include the repo / version or commit hash this was taken from? same thing we do for OZ contracts.

Reply:
@RyanRHall good call. This contract is learnt from [CCIP](https://github.com/smartcontractkit/ccip/blob/263be59233e98347895e73fb3b7c731b1f6a9202/contracts/src/v0.8/ccip/test/WETH9.sol#L4), the original is [here](https://github.com/gnosis/canonical-weth/blob/master/contracts/WETH9.sol). CCIP did some customization and also we **customized** it for our Automation usecase (added the mint function).

Since it is not in its original version, i guess it doesnt qualify to be in the vendor folder any more?

Reply:
oh I see now. Cool not worried about it then! it can stay under `test/`